src-web/actions/constants.js

```diff
@@ -44,25 +46,12 @@ export const REQUEST_STATUS = {
   ERROR: 'ERROR'
 }
 
-export const CONFIG_CONSTANTS = {
```

Comment:
Why remove these constants?  They are required by both kAppNav and Application Navigator.  Seems to me they should remain in kAppNav as an integral part of the implementation.

Reply:
CONFIG_CONSTANTS is not removed but since it's different across Kappnav and Prism, I just moved to its extended file. Extended files are different for kappanv and Prism and later I have import CONFIG_CONSTANTS into Constants.js file